code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -0,0 +1,367 @@
+	for(var/d in attack_dirs)
+		var/turf/E = get_edge_target_turf(src, d)
+		shoot_projectile(E)
+
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
+	for(var/mob/M in range(10,src))
+		if(M.client)
+			M.client.color = rgb(200, 0, 0)
+			spawn(1)
+				M.client.color = initial(M.client.color)
+			shake_camera(M, 4, 3)
+	playsound(get_turf(src),'sound/magic/clockwork/narsie_attack.ogg', 200, 1)
+
+
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/seeking_laser()
+	visible_message("<span class='cult'><font size=5>\"<b>You can't run.</b>\"</font></span>")
+	var/obj/effect/overlay/temp/seeking_beam/B = new(get_turf(src))
+	B.target = target
+	B.firer = src
+
+/obj/effect/overlay/temp/seeking_beam
+	name = "disintegration ball"
+	desc = "Get out of the way!"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "shield-cult"
+	density = 1
+	luminosity = 1
+	var/target
+	var/firer
+	duration = 100
+
+/obj/effect/overlay/temp/seeking_beam/New()
+	..()
+	spawn()
+		while(loc)
+			step_towards(src, target)
+			sleep(1)
+
+/obj/effect/overlay/temp/seeking_beam/process()
+	step_towards(src, target)
+
+/obj/effect/overlay/temp/seeking_beam/Crossed(AM as mob|obj)
+	..()
+	collision(AM)
+
+/obj/effect/overlay/temp/seeking_beam/Bumped(AM as mob|obj)
+	..()
+	collision(AM)
+
+/obj/effect/overlay/temp/seeking_beam/Bump(AM as mob|obj)
+	..()
+	collision(AM)
+
+/obj/effect/overlay/temp/seeking_beam/proc/collision(AM as mob|obj)
+	if(AM == firer)
+		return
+	if(isliving(AM))
+		var/mob/living/M = AM
+		visible_message("<span class='danger'>[M] is disintegrated by the disintegration ball!</span>")
+		M.dust()
+	else if(istype(AM, /obj/item))
+		var/obj/item/I = AM
+		I.burn()
+
+/obj/item/projectile/colossus
+	name ="death bolt"
+	icon_state= "chronobolt"
+	damage = 25
+	armour_penetration = 100
+	speed = 2
+	eyeblur = 0
+	damage_type = BRUTE
+	pass_flags = PASSTABLE
+
+/obj/item/projectile/colossus/on_hit(atom/target, blocked = 0)
+	. = ..()
+	if(istype(target,/turf/)||istype(target,/obj/structure/))
+		target.ex_act(2)
+
+
+/obj/item/device/gps/internal/colossus
+	icon_state = null
+	gpstag = "Angelic Signal"
+	desc = "Get in the fucking robot."
+	invisibility = 100
+
+
+
+//Black Box
+
+/obj/machinery/smartfridge/black_box
+	name = "black box"
+	desc = "A completely indestructible chunk of crystal, rumoured to predate the start of this universe. It looks like you could store things inside it."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_on = "blackbox"
+	icon_off = "blackbox"
+	luminosity = 8
+	max_n_of_items = 10
+	pixel_y = -4
+	use_power = 0
+	var/duplicate = FALSE
+	var/memory_saved = FALSE
+	var/list/stored_items = list()
+
+/obj/machinery/smartfridge/black_box/accept_check(obj/item/O)
+	if(istype(O, /obj/item))
+		return 1
+	return 0
+
+/obj/machinery/smartfridge/black_box/New()
+	..()
+	for(var/obj/machinery/smartfridge/black_box/B in machines)
+		if(B != src)
+			duplicate = 1
+			qdel(src)
+	ReadMemory()
```

Comment:
They'd be overreacting if they did.

Worst case scenario is probably a spell book ending up in there, simply because it's reusable, and those are easily blacklisted.